README.md

```diff
@@ -1,7 +1,5 @@
 # CKEditor 4 - Common Integration Utils
 
-**Work In progress**
```

Comment:
I guess it's still work in progress until we release the package to the `npm`?

Reply:
I suppose no one will notice and we won't accidentally push this note in the npm package.